.github/ISSUE_TEMPLATE/SUPPORT_REQUEST.md

```diff
@@ -0,0 +1,53 @@
+---
+name: Support Request
+about: Report an issue faced while using Hudi
+title: '[SUPPORT]'
+labels: question
+
+---
+
+**_Tips before filing an issue_**
+
+- Have you gone through our [FAQs](https://cwiki.apache.org/confluence/display/HUDI/FAQ)?
+
+- Join the mailing list to engage in conversations and get faster support at dev-subscribe@hudi.apache.org.
+
+- If you have triaged this as a bug, then file an [issue](https://issues.apache.org/jira/projects/HUDI/issues) directly.
+
+**Describe the problem you faced**
+A clear and concise description of the problem.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+1.
+2.
+3.
+4.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Environment Description**
+
+Hudi version :
+
+Spark version :
+
+Hive version :
+
+Hadoop version :
+
+Storage(HDFS/S3/GCS..) :
+
+Running on Docker? :
```

Comment:
Can we give a hint here, like `Running or Docker? : (yes/no)`?

Reply:
Or make it like Storage above, `Running or Docker? (yes/no) :`

Reply:
agree